pyomo/contrib/doe/doe.py

```diff
@@ -1070,127 +792,126 @@ def initialize_jac(m, i, j):
             )
         else:
             model.fim = pyo.Var(
-                model.regression_parameters,
-                model.regression_parameters,
-                initialize=identity_matrix,
+                model.parameter_names, model.parameter_names, initialize=identity_matrix
             )
 
+        # To-Do: Look into this functionality.....
         # if cholesky, define L elements as variables
-        if self.Cholesky_option and self.objective_option == ObjectiveLib.det:
-
-            # move the L matrix initial point to a dictionary
-            if self.L_initial is not None:
-                dict_cho = {
-                    (bu, un): self.L_initial[i][j]
-                    for i, bu in enumerate(model.regression_parameters)
-                    for j, un in enumerate(model.regression_parameters)
-                }
-
-            # use the L dictionary to initialize L matrix
-            def init_cho(m, i, j):
-                return dict_cho[(i, j)]
-
-            # Define elements of Cholesky decomposition matrix as Pyomo variables and either
-            # Initialize with L in L_initial
-            if self.L_initial is not None:
-                model.L_ele = pyo.Var(
-                    model.regression_parameters,
-                    model.regression_parameters,
-                    initialize=init_cho,
-                )
-            # or initialize with the identity matrix
-            else:
-                model.L_ele = pyo.Var(
-                    model.regression_parameters,
-                    model.regression_parameters,
-                    initialize=identity_matrix,
-                )
+        if self.Cholesky_option and self.objective_option == ObjectiveLib.determinant:
+            model.L = pyo.Var(
+                model.parameter_names, model.parameter_names, initialize=identity_matrix
+            )
 
             # loop over parameter name
-            for i, c in enumerate(model.regression_parameters):
-                for j, d in enumerate(model.regression_parameters):
+            for i, c in enumerate(model.parameter_names):
+                for j, d in enumerate(model.parameter_names):
                     # fix the 0 half of L matrix to be 0.0
                     if i < j:
-                        model.L_ele[c, d].fix(0.0)
+                        model.L[c, d].fix(0.0)
                     # Give LB to the diagonal entries
-                    if self.L_LB:
+                    if self.L_diagonal_lower_bound:
                         if c == d:
-                            model.L_ele[c, d].setlb(self.L_LB)
+                            model.L[c, d].setlb(self.L_diagonal_lower_bound)
 
         # jacobian rule
-        def jacobian_rule(m, p, n):
+        def jacobian_rule(m, n, p):
             """
             m: Pyomo model
-            p: parameter
-            n: response
+            n: experimental output
+            p: unknown parameter
```

Comment:
I have generally negative feelings about one-letter var names unless it's for a short loop (e.g., `for i in range(whatever)`). It lowers readability.

Reply:
We're inclined to keep it the way it is since the function is for developers. However, I'd be happy to change it if it's required for the merge.